lte/gateway/c/core/oai/test/mme_app_task/test_mme_procedures.cpp

```diff
@@ -4260,5 +4262,370 @@ TEST_F(MmeAppProcedureTest, TestDuplicateAttach) {
   EXPECT_EQ(mme_state_p->nb_s1u_bearers, 0);
 }
 
+TEST_F(MmeAppProcedureTest, TestNwInitiatedActivateDedicatedBearerRej) {
+  mme_app_desc_t* mme_state_p =
+      magma::lte::MmeNasStateManager::getInstance().get_state(false);
+  std::condition_variable cv;
+  std::mutex mx;
+  std::unique_lock<std::mutex> lock(mx);
+
+  MME_APP_EXPECT_CALLS(3, 1, 1, 1, 1, 1, 1, 1, 0, 1, 3);
+
+  // Constructing and sending Initial Attach Request to mme_app mimicing S1AP
```

Comment:
The initial attach call flow is repeating in several tests. Is it possible to abstract it into a function?